src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/exception/CommunityApiCallError.kt

```diff
@@ -47,6 +41,9 @@ class CommunityApiCallError(val httpStatus: HttpStatus, causeMessage: String, va
         category.contains("Cannot find NSI for CRN: _ Sentence: _ and ContractType".toRegex()) -> {
           "This update has not been possible. This is because there's been a change to the referral in non-structured interventions in nDelius. Ask the probation practitioner if anything has changed with the referral or the person on probation"
         }
+        category.contains("Multiple existing URN NSIs found".toRegex()) -> {
+          "There has been an error during creating the Delius NSI. We are aware of this issue. For follow-up, please contact support"
+        }
         category.contains("Multiple existing matching NSIs found".toRegex()) -> {
           "There has been an error during creating the Delius NSI. We are aware of this issue. For follow-up, please contact support"
         }
```

Comment:
@helennickols "Multiple existing" NSI messages currently say: "there has been an error during creating the Delius NSI". At this point, we would have created multiple NSIs.

Could this say "error during locating" instead?

Reply:
Context: I don't want to give the impression that we failed to create it, as it probably leads to replies like: "but I can see it right there".

Reply:
@helennickols "Multiple existing" NSI messages currently say: "there has been an error during creating the Delius NSI". At this point, we would have created multiple NSIs.

Could this say "error during locating" or similar, instead?

Context: I don't want to give the impression that we failed to create it, as it probably leads to replies like: "but I can see it right there".

It also misleads us when we receive this report from support, as we would be looking at failures to create before looking for duplicates